packages/volto/locales/volto.pot

```diff
@@ -2423,14 +2443,19 @@ msgstr ""
 msgid "No Transactions Selected To Do Undo"
 msgstr ""
 
+#. Default: "No upgrades in this corner. You are up to date. High fives."
+#: components/manage/Controlpanels/Addons/AddonsControlpanel
+msgid "No Upgrades available"
+msgstr ""
+
 #. Default: "No Video selected"
 #: components/manage/Blocks/Video/VideoSidebar
 msgid "No Video selected"
 msgstr ""
 
-#. Default: "No addons found"
+#. Default: "No add-ons found"
 #: components/manage/Controlpanels/VersionOverview
-msgid "No addons found"
+msgid "No add-ons found"
 msgstr ""
```

Comment:
This is a good example of where you it is OK to change the value for `Default` but should revert the change in value for `msgid`.